times.go

```diff
@@ -25,28 +25,50 @@ func Time(then time.Time) string {
 	return RelTime(then, time.Now(), "ago", "from now")
 }
 
-var magnitudes = []struct {
+type Seconds int64
```

Comment:
At first I tried to use `time.Duration`, then users had to new a magnitude using `NewMagnitude(Hour*time.Second, "%d minutes %s", Minute*time.Second)`, and the NewMagnitude function need to do something like `d: d/time.Second`, it's unnecessary and looks strange.
Another reason is for human, second is enough and nanosecond is too much.